start/data/output-file-transfer-via-htcondor.md

```diff
@@ -0,0 +1,80 @@
+
+[title]: - "Output File Transfer via HTCondor"
+
+# Output File Transfer via HTCondor
+
+## Overview
+
+When your OSG Connect jobs run, any output that gets generated is specifically written to 
+the execute node on which the job ran. In order to get access to your output files, a copy of 
+the output must be transferred back to your OSG Connect login node. 
+
+This guide will describe the necessary steps, along with important considerations, for transferring your 
+output files back to your `/home` directory on your OSG Connect login node. 
+
+## Important Considerations
+
+As described in the [Introduction to Data Management on OSG Connect](https://support.opensciencegrid.org/support/solutions/articles/12000002985), 
+any output <1GB should be staged in your `/home` directory. For output files >1GB, 
+please refer to our [Transferring Data with StashCache](https://support.opensciencegrid.org/support/solutions/articles/12000002775) guide.
+
+By default, HTCondor will transfer any files that are generated during the execution of your job(s) 
+back to your `/home` directory, specifically to the directory from which the `condor_submit` 
+command was performed when submitting your job(s). However, this behavior only applies to files 
+in the top-level directory of where your job executes, meaning HTCondor will ignore any files 
+located in subdirectories (because HTCondor ignores subdirectories when determining what needs 
```

Comment:
```suggestion
created in subdirectories of the job's main working directory (because HTCondor ignores subdirectories when determining what needs 
```